Изменения в Лямбда-врн

Primetalk opened the pull request:
Предлагаю обсудить эти изменения и, возможно, что-то изменить в нашем манифесте.

ppopoff committed:
Remove propaganda

We should do no propaganda. We should educate and enlighten the others.
I think that's the crucial concept of our community

ppopoff committed:
Changed some of the principles

Changed their priorities and added an item about our meetings.

ppopoff committed:
Minor changes / numbering

ppopoff committed:
minor fix

я -> е

ppopoff commented:
Какой посыл заключил я.

1. Пропаганда это плохо. Слово имеет негативный оттенок. И означает скорее насильственное вдалбливание идей в головы. Мы просвещаем тех кто хочет просвещаться. И помогаем тем, кто в этом заинтересован. Сообщество должно быть против любой пропаганды, если конечно дело не идет о каких-то идеях типо тех же tagless final.
2. Теперь о пропаганде. Она бывает разная. К нам может прийти блогерКа, авторКа и рассказывать про необходимость феминитивов в языке а так же про то какие мужчины козлы и узурпаторы. Феминизм дело благое (пока он не перерастает в мужененавистничество, как это чаще всего и просиходит), но сообщество это не место для подобного рода пропаганды. То же касается сыроедения, и так далее. Каждый волен заниматься тем что ему по душе. Сообщество не должно быть площадкой для обсуждения этого. И если начинается травля, то имеет смысл указать пропагандисту на то, что он(она)(любой другой гендер) стал инициатором этой травли.
3. Хотя сама травля в сообществе не допустима. Так же как и дискриминация людей по возрасту, гендерной принадлежности, цвету кожи ориентации не допустима. 

Мы можем дискриминировать только идеи и технологии. И если Go -- это это кусок дерьма, значит go это кусок дерьма [спасибо].

С компаниями можно и нужно сотрудничать, если сотрудничество взаимовыгодно. Без этого в современном мире ничего не делается. У нас конечно есть ребята с Пабом и проектором, но они вкладываются там, где можно было бы обойтись без потерь в том числе финансовых.

ppopoff committed:
Changed the audience list

Added a bunch of groups we could be interested in.
Removed a paragraph which could not be defined as a rule or statement.

ppopoff committed:
Update readme.md
